src/components/Layout/UserDropdown/MiniQuotaDisplay/index.tsx

```diff
@@ -0,0 +1,86 @@
+import Infinity from '@app/assets/infinity.svg';
+import { SmallLoadingSpinner } from '@app/components/Common/LoadingSpinner';
+import ProgressCircle from '@app/components/Common/ProgressCircle';
+import type { QuotaResponse } from '@server/interfaces/api/userInterfaces';
+import { Suspense } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+import useSWR from 'swr';
+
+const messages = defineMessages({
+  movierequests: 'Movie Requests',
+  seriesrequests: 'Series Requests',
+});
+
+type MiniQuotaDisplayProps = {
+  userId: number;
+};
+
+const MiniQuotaDisplay = ({ userId }: MiniQuotaDisplayProps) => {
+  const intl = useIntl();
+  const { data } = useSWR<QuotaResponse>(`/api/v1/user/${userId}/quota`, {
+    suspense: true,
+  });
+
+  return (
+    <Suspense fallback={<SmallLoadingSpinner />}>
+      {(data?.movie.limit ?? 0) !== 0 && (data?.tv.limit ?? 0) !== 0 && (
+        <div className="flex">
+          <div className="flex basis-1/2 flex-col space-y-1">
+            <div className="text-sm text-gray-200">
+              {intl.formatMessage(messages.movierequests)}
+            </div>
+            <div className="flex h-full items-center space-x-2">
+              {data?.movie.limit ?? 0 > 0 ? (
+                <>
+                  <ProgressCircle
+                    className="h-8 w-8"
+                    progress={Math.round(
+                      ((data?.movie.remaining ?? 0) /
+                        (data?.movie.limit ?? 1)) *
+                        100
+                    )}
+                    useHeatLevel
+                  />
+                  <span className="text-lg font-bold text-gray-200">
+                    {data?.movie.remaining}/{data?.movie.limit}
+                  </span>
+                </>
+              ) : (
+                <span className="text-sm text-gray-200">
+                  <Infinity className="h-8" />
```

Comment:
Can add `Unlimited` text next to this so that it better matches the mini progress circle with accompanying text?

Reply:
Hm. Same thing here. The icon sizes are different. I think it's clear without accompanying text. I'll see how it looks.

Reply:
I think it'd be nicer to "square" the icon so that it occupies the same space as the progress circle? 🤔  Also, we can add the infinity icon to the actual profile pages, too!